### ISSUE TITLE
User Component Children Prop Not Working

### ISSUE THREAD

**Issue by GraySmith00** [prevwong/craft.js#130]
**Describe the bug**
Children prop for User Components not working as of beta 11 release. (Working in Beta 10)

**To Reproduce**
1. Create a User Button Component:

```jsx
import React from 'react';

export const Button = ({ size, variant, color, children }) => {
  return (
    <button size={size} variant={variant} color={color}>
      {children}
    </button>
  );
};

Button.craft = {
  defaultProps: {
    children: 'CLICK ME',
  },
};

export default Button;
```

**Expected behavior**
Expect to see CLICK ME appear as the child to the button component

**Additional context**
If you change "children" to any other word, "childs" for example, the text appears. 

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| craft.js         | v0.1.0-beta.11 |
| React            | ^16.12.0 | 
| TypeScript       | N/A |
| Browser          | Chrome 85 |
| npm/Yarn         | yarn 1.22.4 | 
| Operating System | OSX |

**Comment by andredovale** [comment]
I'm having the same issue. Trying to render this simple JSON, the "children" prop is ignored:

```javascript
`{
  "canvas-ROOT": {
    "type": "div",
    "props": {},
    "nodes": ["node-TEST"]
  },
  "node-TEST": {
    "type": "p",
    "props": {
      "children": "Lorem ipsum"
    }
  }
}`
```

However, it is not a simple "ignored", because in the attempts I applied here, I found that the "children" prop is converted into an empty `<Fragment>`. I got to this point trying this "dangerous":

```javascript
`{
  "canvas-ROOT": {
    "type": "div",
    "props": {},
    "nodes": ["node-TEST"]
  },
  "node-TEST": {
    "type": "p",
    "props": {
      "dangerouslySetInnerHTML": {
        "__html": "Lorem ipsum"
      }
    }
  }
}`
```

**Comment by RaGreen** [comment]
```
import React from 'react';

const defaultProps = {
    children: 'Text'
}

export const Button = (props) => {
   const { size, variant, color, children } = {
       ...defaultProps,
       ...props
   }

  return (
    <button size={size} variant={variant} color={color}>
      {children}
    </button>
  );
};

Button.craft = {

};

export default Button;
```

Isn't it working this way?

**Comment by Spenc84** [comment]
I am experiencing this problem as well. I haven't had a chance to dig to far into it yet but from what I can tell, Craft appears to be replacing the value of the children prop on every UserComponent and UserElement rendered by \<Frame\> with an empty React fragment. I haven't looked through their code yet, but I am assuming that the children are likely extracted at some point and turned into nodes and then manually injected into place via the DOM. I would guess that some recently introduced bug is preventing that process from working correctly when the value of children is a text string.

That being said, until the bug is found and squashed, the work around would be to pass the text down as a differently named prop and use that as the child of your component instead. For example...

(1) This will work:
`const Text = ( value, ...props ) => <span {...props} children={value} />`

(2) This will not:
`const Text = ( value, ...props ) => <span children={value} {...props} />`

(3) This will work:
`<Text value='Hello World' />`

(4) This will not:
`<Text>Hello World</Text>`

In the last example (4) 'Hello World' would normally be passed down into the Text component as the value of the children prop, but \<Frame\> appears to be swapping that value out with an empty Fragment and then failing to reinsert the text into the DOM later, so your text is lost. Passing the text down as a different prop (like in example 3) will keep your text safe, just keep in mind that the children prop will still have the value <Fragment /> that Craft appears to be giving to it, so doing something like I did in example (2) still will not work since props.children has a defined value.

**Comment by prevwong** [comment]
Should be fixed now, please try again with 0.1.0-beta.13! 😄

**Comment by Spenc84** [comment]
Appears to be working fine now. Thanks.
